projects/potcoin.md

```diff
@@ -0,0 +1,20 @@
+## Overview
+![ logo](logo/potcoin.png)  
+Name: Potcoin  
+[Website](http://www.potcoin.com)  
+Year founded: 2014   
+Currency: [POT](https://coinmarketcap.com/currencies/potcoin/)  
+## Staff
+Co-founder: [Nick Iversen aka smokemon514](../people/nick_iversen.md)   
+Co-founder: [Joel Yaffe aka Mr. Jones](../people/mr_jones.md)     
+Dev: [Hasoshi](../people/hasoshi.md) 
+## Business Model
+PotCoin is a peer-to-peer cryptocurrency which exists with the aim of becoming the standard form of payment for the legalized cannabis industry. PotCoin is an open source software project released under the MIT/X11 license and was technically nearly identical to Litecoin until August 23, 2015, when Potcoin changed to POSV similar to Reddcoin.
```

Comment:
we don't have reddcoin yet. However, I can make a link and when we will do a reddcoin project, it will be linked automatically.